byzcoin/bcadmin/clicontracts/deferred.go

```diff
@@ -85,14 +85,13 @@ func DeferredSpawn(c *cli.Context) error {
 		},
 	}
 
-	ctx := byzcoin.ClientTransaction{
-		Instructions: []byzcoin.Instruction{
-			{
-				InstanceID:    byzcoin.NewInstanceID(d.GetBaseID()),
-				Spawn:         &spawn,
-				SignerCounter: []uint64{counters.Counters[0] + 1},
-			},
-		},
+	ctx, err := cl.CreateTransaction(byzcoin.Instruction{
```

Comment:
Do I understand correctly that you're calling out to a websocket within a contract? If yes, then this should be removed and replaced with `NewClientTransaction`, as you should never depend on external states within a contract!